packages/block/package.json

```diff
@@ -70,8 +69,5 @@
     "typedoc-plugin-markdown": "^2.2.17",
     "typescript": "^3.4.3",
     "typestrict": "^1.0.2"
-  },
-  "publishConfig": {
-    "directory": "src"
```

Comment:
We can't do the direct typescript code linking yet without handling the browser tests in a cross-package scope.